release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+* link:https://github.com/kubernetes/website/blob/master/docs/concepts/api-extension/apiserver-aggregation.md[Kubernetes aggregation layer]
+
+[[ocp-37-master-prometheuh-endpoint-coverage]]
+==== Master Prometheus Endpoint Coverage
+
+Prometheus endpoint logic was added to upstream components so that monitoring
+and health indicators can be added around deployment configurations.
+
+[[ocp-37-end-to-end-provider-integration]]
+=== End-to-end Provider Integration
+
+[[ocp-37-deploying-cns-for-opsenshift-36]]
+==== Deploying Container-Native Storage for OpenShift Container Platform 3.6
+
+The
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/[Deploying
+Container-Native Storage for OpenShift Container Platform 3.6] guide describes
+the prerequisites and provides step-by-step instructions to deploy
+container-native storage (Red Hat Gluster Storage 3.3) with {product-title}.
+
+[[ocp-37-openshift-for-ops-test-drive]]
+==== OpenShift for Ops Test Drive
+
+The link:https://www.redhat.com/en/engage/openshift-storage-testdrive-20170718[OpenShift for Ops Test Drive] provides system administrators with a no-cost, hands-on lab experience to evaluate OpenShift in a fully functional cloud-based environment.
+There is a self-paced, hands-on lab environment, in which students learn:
+
+* How to install and configure OpenShift, including log and metrics aggregation and external authentication.
+* How to install and configure CNS (Red Hat Gluster Storage) inside {product-title}.
+* How to perform basic administrative and maintenance operations.
+* Basic application life cycle with health and readiness monitoring.
+
+[[ocp-37-installation]]
+=== Installation
+
+[[ocp-37-migrate-etcd-before-upgarde]]
+==== Migrate etcd Before OpenShift Container Platform 3.7 Upgrade
+Starting in {product-title} 3.7, the use of the etcd3 v3 data model is required.
+
+{product-title} gains performance improvements with the v3 data model. In order
+to upgrade the data model, an embedded etcd configuration option in no longer
+allowed. Embedded is not co-located and mainly used in single-master
+deployments. Migration scripts will convert the v3 data model and allow you to
+move an embedded etcd to an external etcd either on the same host or a different
+host than the masters. In addition, there is a new scale up ability for etcd
+clusters.
+
+See
+xref:../install_config/upgrading/migrating_embedded_etcd.adoc#install-config-upgrading-etcd-data-migration[Migrating
+Embedded etcd to External etcd] for more information.
+
+[[ocp-37-modular-installer]]
+==== Modular Installer to Allow Playbooks to Run Independently
+
+The installer has been enhanced to allow administrators to install specific
+components. By breaking up the roles and playbooks, there is better targeting of
+ad hoc administration tasks.
```

Comment:
I understand that this might be a header section, but, do we have docs that have examples of this (whats discussed)? What admin tasks does this now cover? How does this help users? 

The below items on phases might be what you're trying to say, but this section does not connect well to that, nor does it (in the following sections) show or demonstrate why this is important/useful. 

Do we have a section of docs that we could link to that might demonstrate this better?

Reply:
Since the End-to-end Provider Integration and subsections were more about helpful resources that are now available and not 3.7 offerings, I just removed it to prevent confusion.